config/sidekiq.yml

```diff
@@ -20,6 +20,9 @@
   weekly_digest_initiator:
     cron: '30 8 * * 6 Europe/London' # every Saturday at 8:30am
     class: WeeklyDigestInitiatorWorker
+  historical_data_deletion:
+    cron: '0 0 * * * Europe/London' # every day at midnight
```

Comment:
I chose a time out of hours as I didn't deem this worker to be important enough to warrant running in hours considering it could also effect performance. 

> I wonder if we'd do want to do this during the working day so we get an alert if it fails

We'd also still get an alert the next day if it failed? 

I agree if it's slow we should optimise it but I don't think deleting more frequently would help much due to the fact the bulk of it's load is created with a 30 minutes window (all digest subscriber runs).

I'm still not opposed to moving in hours if you feel strongly?

Reply:
I'd start with placing it there until you have evidence that it shouldn't be run in hours. There are lots of things scheduled out of hours at some point in time that you can encounter and you have no idea if they are safe to run in hours or not as no-one ever tried.

Reply:
Ah okay, that makes sense! I've set it to midday.